homeassistant/components/downloader/services.yaml

```diff
@@ -27,3 +27,11 @@ download_file:
       default: false
       selector:
         boolean:
+    timeout:
+      name: Timeout
+      description: Timeout in seconds for http call
+      default: 10
+      selector:
+        number:
+          min: 10
+          max: 60
```

Comment:
Is there a reason the timeout bounds are set to 10-60? I think it makes more sense to have the lower bound be 1, or even 0. For the upper bound, perhaps 300 seconds (5 minutes), which matches many of the other time limits for other things around Home Assistant such as light transitions.